webpack.config.cjs

```diff
@@ -229,7 +229,7 @@ const templateContent = ({ htmlWebpackPlugin }) => {
 const editorAndDashboard = {
   ...sharedConfig,
   entry: {
-    'edit-story': './assets/src/edit-story/index.js',
+    'edit-story': './packages/wp-story-editor/src/index.js', // @todo Change key name to wp-story-editor
```

Comment:
Is there still a todo here? Seems easy enough to fix...

Reply:
The plan is to do this in a separate PR targeting this branch as the diff is quite big.

Reply:
Yes, I was thinking to update all usage of `edit-story` to `story-editor` / `wp-story-editor` all together wherever makes sense. For example karma configuration file names and configuration paths. But this todo alone can be removed with few changes, done in https://github.com/google/web-stories-wp/pull/8636